data/priority_area.geojson

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
@planemad We could rename the file to `priority_areas.geojson` as this is a `FeatureCollection` of multiple priority areas.